README.md

```diff
@@ -111,6 +111,7 @@ If you are looking for the codebase of the current production version of GATK, p
             --conf spark.yarn.executor.memoryOverhead=600
         ```
 
+    * You can also omit the "--num-executors" to enable [dynamic allocation](https://spark.apache.org/docs/latest/job-scheduling.html#dynamic-resource-allocation) if you configure the cluster properly (see the Spark website for instructins).
```

Comment:
I don't think so. The number of executors is the the thing that's dynamic, not the size of an executor.